why_angular_2.md

```diff
@@ -1,38 +1,52 @@
+One of Angular's biggest advantages is how its structure supports learning and understanding. Developers new to the framework can tackle independent aspects of the framework, build an understanding, and expand their understanding to the next concept when they're ready. Teams that keep their code structure and guidelines close to the [Angular Styleguide](https://angular.io/guide/styleguide) will see that new developers can more quickly spin up and work on complex engineering problems using this base documentation to accelerate their understanding of your codebase.
 
-Compared to other popular web frameworks, Angular is 
+It can't be understated that many other frameworks require you to shift mental models first before being able to dive in. With Angular's structure, the code is broken down to provide a separation of concerns ensuring that code is broken apart sensibly. Teams will need to maintain this separation of concerns as their project expands, but many other frameworks don't help teams organize their projects to this degree from the get-go.
 
-### TypeScript
+Automation and generation have also been leveraged to quickly spin up new files and simplify the bootstrapping work needed to create new features. As we'll see in later chapters, Schematics in the Angular CLI also go a long way to help newbies feel comfortable creating new components, services, and features.
 
-Angular 2+ was written in TypeScript, a superset of JavaScript that implements many new ES2016+ features, and was since its inception. 
+While there are many aspects of Angular that provide an accessible entry for new developers, understanding the intricacies of bundling files, dependency injection, and many other concepts remains challenging and will take time for teams to learn. Angular helps to isolate these problems allowing for focused review and incrementally improving the effort required to globally refactor challenging problems.
 
-By focusing on making the framework easier for computers to process, Angular allows for a much richer development ecosystem. Programmers using sophisticated text editors \(or IDEs\) will notice dramatic improvements with auto-completion and type suggestions. These improvements help to reduce the cognitive burden of learning Angular. Fortunately for traditional ES5 JavaScript programmers this does _not_ mean that development must be done in TypeScript or ES2015: programmers can still write vanilla JavaScript that runs without transpilation.
+### Well Leveraged TypeScript
 
-### Familiarity
+Angular 2+ was written in TypeScript, a typed superset of JavaScript that implements many modern EcmaScript features. Programmers using sophisticated text editors \(or IDEs\) will notice dramatic improvements with auto-completion and type suggestions. These improvements help to reduce the cognitive burden of learning Angular.
```

Comment:
> Angular ~~2+~~

This is very clear though. I very good example of how the rest of the graphs should read.